tests/cppunit/types/list_test.cc

```diff
@@ -162,6 +177,89 @@ TEST_F(RedisListTest, Range) {
   list_->Del(key_);
 }
 
+TEST_F(RedisListLPosTest, Pos) {
```

Comment:
A small change here: `RedisListLPosTest` -> `RedisListTest` (in order to not introduce a new group of tests).